go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/mattermost/mattermost-plugin-todo
 go 1.13
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.20.0
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200527124843-dea705969ca4
```

Comment:
I understand this is targetting certain commit or branch. Couldn't we target a proper version? (5.23? 5.24?)

Reply:
Sure, we will target 5.24 when it is cut.

Reply:
https://github.com/mattermost/mattermost-server/releases/tag/v5.24.0-rc1 can be used